src/API/OnboardingProfile.php

```diff
@@ -209,6 +209,20 @@ public static function get_profile_properties() {
 				'readonly'          => true,
 				'validate_callback' => 'rest_validate_request_arg',
 			),
+			'skipped'             => array(
+				'type'              => 'boolean',
+				'description'       => __( 'Whether or not the profile was skipped.', 'woocommerce-admin' ),
+				'context'           => array( 'view' ),
+				'readonly'          => true,
+				'validate_callback' => 'rest_validate_request_arg',
+			),
+			'step'                => array(
```

Comment:
Is the "skip" option only available on the "Store Details" step?  If so, I don't think we need to save the "step" data.

Reply:
I'm not sure if I understood correctly, I'm using the `step` to take the user back to the OBW step where they left off. If you think that this is not necessary just let me know and I'll change it 🙂

Reply:
I think we won't need it since the "Skip" button is only added to the "Store Details" step in https://github.com/woocommerce/woocommerce-admin/pull/4771/files

We can probably safely remove any reference to this saved step and "`lastStep` since it will always be the first step if they skip.

Reply:
Just a heads up (not sure if you're still working on this) but this can be removed still.

Reply:
This property still needs to be removed.